### PR TITLE
Change package name for module mixed-format/trino

### PR DESCRIPTION
**Pull request by zhoujinsong**
## Why are the changes needed?

Change package name to `org.apache.amoro` for module mixed-format/hive.

## Brief change log

- Change package name

## How was this patch tested?

- [ ] Add some test cases that check the changes thoroughly including negative and positive cases if possible

- [ ] Add screenshots for manual tests if appropriate

- [x] Run test locally before making a pull request

## Documentation

- Does this pull request introduce a new feature? (no)
- If yes, how is the feature documented? (not applicable)

### CODE REVIEW

**Review by xleoken — APPROVED** [review]

File: mixed-format/trino/src/test/java/org/apache/amoro/trino/mixed/ArcticQueryRunner.java [hunk 1]
```diff
@@ -16,16 +16,16 @@
  * limitations under the License.
  */
 
-package com.netease.arctic.trino.arctic;
+package org.apache.amoro.trino.mixed;
```
Comment (zhongqishang) [inline mixed-format/trino/src/test/java/org/apache/amoro/trino/mixed/ArcticQueryRunner.java:19]: The package name confuses me. This package is test for mixed hive format?

**Review by zhongqishang — COMMENTED** [review]
Thanks for your contribution, I left some comments.

File: mixed-format/trino/src/test/java/org/apache/amoro/trino/mixed/ArcticQueryRunner.java (hunk 1 above)
Comment (zhoujinsong) [reply mixed-format/trino/src/test/java/org/apache/amoro/trino/mixed/ArcticQueryRunner.java:19]: Not just mixed-hive format, including both mixed-iceberg and mixed-hive.

File: mixed-format/trino/src/main/java/org/apache/amoro/ArcticErrorCode.java [hunk 2]
```diff
@@ -16,7 +16,7 @@
  * limitations under the License.
  */
 
-package com.netease.arctic;
+package org.apache.amoro;
```
[the remark this answers is not in the document]
Comment (zhoujinsong) [reply mixed-format/trino/src/main/java/org/apache/amoro/ArcticErrorCode.java:19]: This PR will not change the package of the class but just rename the package name.

**Review by zhongqishang — APPROVED** [review]
LGTM.

**Comment by zhoujinsong** [comment]
> LGTM.

Thanks for the review.

**Pull request merged**